.travis.yml

```diff
@@ -55,4 +54,4 @@ script:
   - bundle exec rails db:structure:load
   - bundle exec rails db:seed
   - rake ci
-  - bundle exec codeclimate-test-reporter
+  - CODECLIMATE_REPO_TOKEN=861a17b4c76ddedf9972cdaca2771f9ab0c99f4ce92298fa965d25a4be55bdbe bundle exec codeclimate-test-reporter
```

Comment:
If you mean this:
`addons:
  code_climate:
    repo_token:`
Well, my report was not formed with this setting, rather, report is went, but codeclimate did not show this report, that why i said:

> will see what can i do, I understand and agree that it's ugly